src/libsystemd/sd-device/device-enumerator.c

```diff
@@ -273,6 +274,14 @@ _public_ int sd_device_enumerator_allow_uninitialized(sd_device_enumerator *enum
 
         return 1;
 }
+_public_ int sd_device_enumerator_always_match_parent(sd_device_enumerator *enumerator)
+{
```

Comment:
Maybe, better to name `sd_device_enumerator_add_all_parents()` ?

Please do not break line after `)`, that is,
```c
_public_ int sd_device_enumerator_add_all_parents(sd_device_enumerator *enumerator) {
```